python/gubbins/common.py

```diff
@@ -325,6 +326,13 @@ def parse_and_run(input_args, program_description=""):
             printer.print("\nReinserting gaps into the alignment...")
             shutil.copyfile(base_filename + ".start", gaps_alignment_filename)
             reinsert_gaps_into_fasta_file(joint_sequences_filename, gaps_vcf_filename, gaps_alignment_filename)
+            print("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
+            print("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
+            print(gaps_alignment_filename)
+            print("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
+            print("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
+            # if input_args.seq_recon == "raxmlng":
+            #     remove_internal_nodes_from_alignment(gaps_alignment_filename)
             if not os.path.exists(gaps_alignment_filename) \
```

Comment:
Probably don't need this debugging statement?

Reply:
Removed